### PR TITLE
increase alert cleared timeout for test_ceph_health and alert collection for measure_noobaa_ns_target_bucket_deleted

### PR DESCRIPTION
**Pull request by fbalak**
- Fixes https://github.com/red-hat-storage/ocs-ci/issues/8138
- Fixes https://github.com/red-hat-storage/ocs-ci/issues/9509

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/35465/
Check for clearing alerts passed correctly

File: tests/functional/monitoring/prometheus/alerts/test_ceph.py [hunk 1]
```diff
@@ -87,6 +87,10 @@ def test_ceph_health(measure_stop_ceph_osd, measure_corrupt_pg, threading_lock):
     utilization.
     """
     api = prometheus.PrometheusAPI(threading_lock=threading_lock)
+    # the time to wait is increased because it takes more time for Ceph
+    # cluster to resolve its issues
+    health_wait = 420
```
Comment (DanielOsypenko) [inline tests/functional/monitoring/prometheus/alerts/test_ceph.py:92]: this value is set to 420, but these are minutes. 420/60 = 7, total 7 hours of wait .

```
 api.check_alert_cleared(
        label=target_label,
        measure_end_time=stop_time,
        time_min=health_wait,
    )
```

**Review by DanielOsypenko — CHANGES_REQUESTED** [review]
I assume Cluster issues that we are checking are not covering potential bug

**Review by DanielOsypenko — APPROVED** [review]

File: tests/functional/monitoring/prometheus/alerts/test_ceph.py (hunk 1 above)
Comment (DanielOsypenko) [reply tests/functional/monitoring/prometheus/alerts/test_ceph.py:92]: sorry, ignore this.
time_min - it looks more like time in minutes than minimal time, which I expected to see as a min_time

As an enhancement, suggest to change this parameter name. Up to you, in this PR or in another.

**Review by dahorak — APPROVED** [review]

**Pull request merged**